### PR TITLE
[10.0] fix bug on negative invoice payments

### PR DESCRIPTION
**Pull request by sergiocorato**
fix https://github.com/odoo/odoo/issues/19280

### CODE REVIEW

**Comment by sergiocorato** [comment]
@odoo-italia to verify

File: account_payment_negative_invoice/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+               "account_invoice_negative",
+    "author": "Efatto.it di Sergio Corato, Odoo Community Association (OCA)",
+    "maintainers": ["sergiocorato"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "account",
+    ],
+    "data": [
```
Comment (Nikul-Chaudhary) [inline account_payment_negative_invoice/__manifest__.py]: @sergiocorato If no data then remove it

Comment (sergiocorato) [reply account_payment_negative_invoice/__manifest__.py]: @Nikul-Chaudhary done

**Comment by sergiocorato** [comment]
Close for inactivity

**Pull request closed without merging**